msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1454,6 +1452,17 @@ When you use these install and uninstall events, there are some instances where
 
 It's important to know that new events can be added anytime in the future and your bot begins to receive them. So you must design for the possibility of receiving unexpected events. If you're using the Bot Framework SDK, your bot automatically responds with a 200 – OK to any events you don't choose to handle.
 
+## Handling errors in conversation events
+
+When the bot encounters an error while handling different events or activities, it shouldn't post random messages directly to the conversation, following is an example:
```

Comment:
Don't say "random"-- from the bot developer perspective they may be posting a descriptive error message.

Reply:
@v-sdhakshina - Please update like below:
"When the bot encounters an error while handling different events or activities, it better to send appropriate messages directly to the conversation, following is an example:"

Reply:
Updated the document as suggested.